.circleci/config.yml

```diff
@@ -1,35 +1,85 @@
-version: 2
+version: 2.1
+
+orbs:
+  zulip: ponylang/zulip@1
+
 jobs:
-  ponyup:
+  x86-64-unknown-linux-nightly:
     docker:
-      - image: ponylang/ponyc:latest
+      - image: ponylang/ponyc:alpine
     steps:
+      - run: apk add --update py-pip
+      - run: pip install cloudsmith-cli
       - checkout
+      - run: bash .ci-scripts/x86-64-unknown-linux-nightly.bash ${CLOUDSMITH_API_KEY}
+      - zulip/status
 
+  shellcheck-init-script:
+    docker:
+      - image: ponylang/ponyc:latest
+    steps:
+      - checkout
       - run: apt-get update -y && apt-get install -y shellcheck
       - run: shellcheck ponyup-init.sh
 
+  vs-ponyc-release:
+    docker:
+      - image: ponylang/ponyc:release
+    steps:
+      - checkout
       - run: .ci-scripts/install-openssl.sh
+      - run: make test config=debug
+      - run: .ci-scripts/test-ponyup-ponyc.sh
+      - zulip/status:
+          fail_only: true
+
+  vs-ponyc-master:
+    docker:
+      - image: ponylang/ponyc:latest
+    steps:
+      - checkout
+      - run: make test config=debug
```

Comment:
if there arent "config=debug" and "config=release" jobs for each, i'd lean towards not doing config=debug here and on the other job as well.

Reply:
you makefile requires `ssl` to be set but this isn't using that.